contrib/redhat/collectd.spec

```diff
@@ -2084,7 +2100,7 @@ Collectd utilities
 	%{?_with_capabilities} \
 	%{?_with_ceph} \
 	%{?_with_cgroups} \
-	%{?_with_check_uptime} \
+	%{?_with_check_presence} \
```

Comment:
Why is check_uptime removed here?

Reply:
That was an edit that should have been a duplicate line, I've just fixed this one.